src/Illuminate/Database/Query/Processors/SqlServerProcessor.php

```diff
@@ -18,8 +18,7 @@ class SqlServerProcessor extends Processor
     public function processInsertGetId(Builder $query, $sql, $values, $sequence = null)
     {
         $query->getConnection()->insert($sql, $values);
-
-        $id = $query->getConnection()->getPdo()->lastInsertId();
+        $id = $query->selectRaw("SCOPE_IDENTITY() AS lastId")->first()->lastId; 
```

Comment:
styleci's pointed out there's trailing whitespace on the end of this line - please remove it ;)